### ISSUE TITLE
Can't see HTTP response after PUT/PATCH/POST

### ISSUE THREAD

**Issue by jtuliani** [projectkudu/AzureResourceExplorer#202]
When I perform a PUT/PATCH/POST, it would be really useful to see the HTTP response.  HTTP status and response body for sure, ideally also the headers although I appreciate that's perhaps more of a 'power user' feature that perhaps belongs under /raw

**Comment by bala16** [comment]
Yes. the main site is kept simple to avoid clutter. this might be better suited to /raw if we decide to do this.
